lib/models/requestResponse.js

```diff
@@ -1,12 +1,116 @@
 'use strict';
 
-const
-  assert = require('../utils/assertType');
+const assert = require('../utils/assertType');
 
 const
```

Comment:
I know this may be outside the scope of this PR but can we get ride of these "hidden/protected" properties ?

Reply:
This can be debated I guess, probably in one of our product workshop.

Reply:
I will fill a ticket so